.gitlab-ci.yml

```diff
@@ -103,6 +103,12 @@ Setup:Review:
   image: dockerhub.ebi.ac.uk/ensembl-web/deploy-tools:latest
   environment:
     name: development
+  before_script:
+    - kubectl config use-context ${AGENT}
+    - kubectl config set-context --current --namespace=${NS}
+  variables:
+    AGENT: ${REVIEW_AGENT}
+    NS: ${CI_COMMIT_REF_SLUG}
   script:
     - kubectl create namespace ${CI_COMMIT_REF_SLUG} --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
It's better to remove this line: ens-client will need to create review namespace anyway, plus here it's run in wrong namespace (should to be dev ns intead of review ns).

Reply:
Thank you @veidenberg , Is it ok if I delete it and revert back the variables to point to dev as well?
```
variables:
    AGENT: ${DEV_AGENT}
    NS: ${DEV_NS}
```

Reply:
Leave the variables as is (the ns/agent needs to be pointing to review for creating the configmap).

Reply:
Thank you for the clarifications @veidenberg , I removed that line